osu.Game/Overlays/UserButton.cs

```diff
@@ -0,0 +1,150 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input;
+using osu.Game.Graphics;
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework;
+using osu.Framework.Graphics.Primitives;
+
+namespace osu.Game.Overlays
+{
+    class UserButton : Container
+    {
+        public const float WIDTH = 60;
+
+        public Sprite Image

```

Comment:
it's good to group these with the thing they expose.
eg: this would be just below the definition for ```DrawableAvatar```